src/components/my-map/my-map.js

```diff
@@ -1,26 +1,29 @@
 import './my-map.scss';
+import './configuration/evt-widget-config-marker-color/evt-widget-config-marker-color';
```

Comment:
If you want to group these two into an angular module, that's done outside of either component. And adding that `my-map` module as a dependency of the `'myModule.components'`:

```javascript
export default angular.module('myModule.components', [
  'myModule.components.myMap'
])
```

Though, the point of this repo was to be as simple as possible and not complicate.